lua/caramba/commands.lua

```
@@ -1272,6 +1362,66 @@ Provide specific, actionable feedback with examples where applicable.
+        if freshness.total_missing > 10 then
+          vim.notify(string.format("  ... and %d more", freshness.total_missing - 10), vim.log.levels.ERROR)
+        end
+      end
+      
+      vim.notify("\nRun :AIRefreshIndex to update the index", vim.log.levels.INFO)
+    end
+  end, {
+    desc = "AI: Check index freshness",
+  })
+  
+  vim.api.nvim_create_user_command("AIToggleAutoIndex", function()
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)